packages/api/src/controllers/region.js

```diff
@@ -0,0 +1,57 @@
+import Router from 'express/lib/router'
+import { authMiddleware } from '../middleware'
+
+const app = Router()
+
+const getOrchestrators = async (req, res, next) => {
```

Comment:
I don't think we need `req.getOrchestrators()` anywhere here. Maybe we will once we figure out how each region is individually updating the global O list, but this set of endpoints shouldn't need it for anything. It's all about just getting updated externally.